packages/component-library/assets/global.styles.css

```diff
@@ -133,48 +57,42 @@ a:hover {
   margin-bottom: 12px;
 }
 
-h1,
-.h1 {
```

Comment:
@jaronheard, I did a regex search excluding `/h1` and `<h1` and didn't find anything that looked like an h1 class. I repeated this for the others as well so I feel ok about removing this but I just wanted to check that there isn't a use case I'm missing.

Reply:
@Dianna I don't know of other use cases that you'd be missing